x/amm/spec/02_mechanism.md

```diff
@@ -0,0 +1,101 @@
+# Mechanism
+
+The Elys Network AMM module implements an Automated Market Maker (AMM) using various types of liquidity pools. It supports the following types of pools:
+
+1. **AMM Pools**: Non-oracle pools with liquidity centered around a specific spot price, designed for assets with significant price variation.
+2. **Oracle Pools**: Oracle pools with liquidity centered around an oracle price, designed for assets with stable prices.
+
+This section explains the fundamental mechanism of the AMM module and provides an overview of how the module's code is structured to support both types of pools.
+
+## Pool
+
+### Creation of Pool
```

Comment:
is it possible to add more detail about oracle and non-oracle pool equations ?

Reply:
what if in oracle pool, liquidity tokens lose balance w.r.t oracle prices ?

Reply:
@amityadav0 I added some additional content to cover the oracle pool mechanism: https://github.com/elys-network/elys/pull/640/commits/0efeecc1ebcbe0b79da0d41f731820ad742e3c1c